org.eclipse.m2e.tests/src/org/eclipse/m2e/tests/BuildPathManagerTest.java

```diff
@@ -319,7 +319,7 @@ public void testClasspathOrderWorkspace001() throws Exception {
     Set<Artifact> artifacts = getMavenProjectArtifacts(project1);
     assertEquals(4, artifacts.size());
     Artifact a1 = artifacts.iterator().next();
-    assertEquals(project2.getFolder("target/classes").getLocation().toFile(), a1.getFile());
+    assertEquals(project2.getFolder("pom.xml").getLocation().toFile(), a1.getFile());
```

Comment:
Folder?

Reply:
Forgot to change that. Thanks.
But since this is a resource-handle operation this obviously did not broke the test.